packages/vx-event/src/localPointGeneric.ts

```diff
@@ -0,0 +1,33 @@
+import { Point } from '@vx/point';
+import { isSVGElement, isSVGGraphicsElement, isSVGSVGElement } from './typeGuards';
+import getXAndYFromEvent from './getXAndYFromEvent';
+
+export default function localPoint(node: Element, event: MouseEvent | TouchEvent) {
```

Comment:
should update all of the event types to also accept `React.MouseEvent | React.TouchEvent`